slug_test.go

```diff
@@ -1088,8 +1122,8 @@ func assertArchiveFixture(t *testing.T, slug *bytes.Buffer, got *Meta) {
 		}
 
 		if hdr.Name == "example.tf" {
-			if hdr.Typeflag != tar.TypeSymlink {
-				t.Fatalf("expected symlink file 'example.tf'")
+			if hdr.Typeflag != tar.TypeReg {
+				t.Fatalf("expected symlink to be dereferenced 'example.tf'")
```

Comment:
I feel like this change is representative of a change in behavior: When `dereference` is true, the godoc says "symlinks with a target outside of the src directory will be dereferenced." but it now appears that symlinks within the src directory are dereferenced as well.

Reply:
In this case no, the previous test was incorrect. `example.tf` lies outside each `src` directory used in test, therefore it should be dereferenced.

Reply:
Wow!